src/api-docs.js

```diff
@@ -939,6 +939,110 @@ const apiDoc = {
         },
       },
     },
+    '/wallet/create-nft': {
```

Comment:
wouldn't a `POST` or `PUT` on `/wallet/nft` already indicate a creation?

Reply:
I agree with you, it's redundant. But I think we should keep it anyway to keep the standard because we already have `/wallet/create-token`. What do you think?

Reply:
I think that there are other paths that could be more clean and easier to understand. But we should keep the standard until we decide we need a refactor.